.github/workflows/release.yml

```diff
@@ -64,5 +64,29 @@ jobs:
              npm whoami
              npm publish --access public
         env:
-          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
         working-directory: ${{ github.workspace }}/src/shell/js/composeui-node-launcher/
+
+  deploy_to_nuget:
```

Comment:
I know it's picky but it's a little bit inconsistent naming:
deploy-npm vs deploy_to_nuget

Reply:
Ah yeah in the original version I made I named both to matchin patterns but I didn't change in this final version, I'll adjust :)